src/components/route/addBook.jsx

```diff
@@ -22,15 +31,17 @@ const BookForm = ({ addBook }) => {
       <form onSubmit={handleSubmit}>
         <input type="text" className="book-title" placeholder="Book title" value={title} onChange={(e) => setTitle(e.target.value)} />
         <input type="text" placeholder="Book author" className="book-author" value={author} onChange={(e) => setAuthor(e.target.value)} />
+        <input
+          type="text"
+          placeholder="Category"
+          value={category}
+          onChange={(e) => setCategory(e.target.value)}
+        />
         <button type="submit">Add Book</button>
```

Comment:
- [ ] onClick event handler should be implemented on this button.The event handler should add a book to the books array with attribute title and author. I think you need to fix this since it is part of the requirements. Check screenshot below for reference :+1: 
![Screenshot from 2023-09-14 10-27-43](https://github.com/misikir21/bookstore/assets/63087491/6223ba14-6368-48c9-9782-8f67d2ccf48e)